src/ApiService/ApiService/onefuzzlib/WebhookOperations.cs

```diff
@@ -1,125 +1,135 @@
-        }
-        else
-        {
-            await _queue.QueueObject("webhooks", obj, StorageType.Config, visibilityTimeout: visibilityTimeout);
-        }
-    }
-
-    private void QueueObject(string v, WebhookMessageQueueObj obj, StorageType config, int? visibility_timeout)
-    {
-        throw new NotImplementedException();
-    }
-
-    public IAsyncEnumerable<WebhookMessageLog> SearchExpired()
-    {
-        var expireTime = (DateTimeOffset.UtcNow - TimeSpan.FromDays(EXPIRE_DAYS)).ToString("o");
-
-        var timeFilter = $"Timestamp lt datetime'{expireTime}'";
-        return QueryAsync(filter: timeFilter);
-    }
-}
-
-
-public interface IWebhookOperations
-{
-    Async.Task SendEvent(EventMessage eventMessage);
-}
-
-public class WebhookOperations : Orm<Webhook>, IWebhookOperations
-{
-    private readonly IWebhookMessageLogOperations _webhookMessageLogOperations;
-    private readonly ILogTracer _log;
-    public WebhookOperations(IStorage storage, IWebhookMessageLogOperations webhookMessageLogOperations, ILogTracer log)
-        : base(storage, log)
-    {
-        _webhookMessageLogOperations = webhookMessageLogOperations;
-        _log = log;
-    }
-
-    async public Async.Task SendEvent(EventMessage eventMessage)
-    {
-        await foreach (var webhook in GetWebhooksCached())
-        {
-            if (!webhook.EventTypes.Contains(eventMessage.EventType))
-            {
-                continue;
-            }
-            await AddEvent(webhook, eventMessage);
-        }
-    }
-
-    async private Async.Task AddEvent(Webhook webhook, EventMessage eventMessage)
-    {
-        var message = new WebhookMessageLog(
-             EventId: eventMessage.EventId,
-             EventType: eventMessage.EventType,
-             Event: eventMessage.Event,
-             InstanceId: eventMessage.InstanceId,
-             InstanceName: eventMessage.InstanceName,
-             WebhookId: webhook.WebhookId
-            );
-
-        var r = await _webhookMessageLogOperations.Replace(message);
-        if (!r.IsOk)
-        {
-            var (status, reason) = r.ErrorV;
-            _log.Error($"Failed to replace webhook message log due to [{status}] {reason}");
-        }
-    }
-
-
-    //todo: caching
-    public IAsyncEnumerable<Webhook> GetWebhooksCached()
-    {
-        return QueryAsync();
-    }
-
-}
+using Microsoft.OneFuzz.Service.OneFuzzLib.Orm;
+using ApiService.OneFuzzLib.Orm;
+using System.Text.Json;
+using System.Security.Cryptography;
+using System.Net;
+
+namespace Microsoft.OneFuzz.Service;
+
+public interface IWebhookOperations
+{
+    Async.Task SendEvent(EventMessage eventMessage);
+    Async.Task<Webhook?> GetByWebhookId(Guid webhookId);
+    Async.Task<bool> Send(WebhookMessageLog messageLog);
+}
+
+public class WebhookOperations : Orm<Webhook>, IWebhookOperations
+{
+
+    // Needs to eventually pull from global __version__

```

Comment:
We should create an issue to track this

Reply:
let's use an app setting for this. i am adding this line in ServiceConfiguration.cs in my current PR
![image](https://user-images.githubusercontent.com/244326/165124334-99d23a3d-9d23-4d04-a2ea-9948c544bf0a.png)

We will need to update the build and the deployment to set this up correctly 
CC: @AdamL-Microsoft

Reply:
grab th vrsion from Config.OnefuzzVerson

Reply:
Done!